lib/watcher/appmetrics.js

```diff
@@ -0,0 +1,37 @@
+exports.worker = function(handle) {
+  var appmetrics = require('appmetrics');
+  var monitoring = appmetrics.monitor();
```

Comment:
Are there any features you can disable in appmetrics to reduce overehead? IIRC, appmetrics defaults to trying to make outgoing MQTT calls, perhaps those can be disabled. I'm just worried that appmetrics is doing more than we need it to, and I know many of its features can be disabled.